dso-l2/src/main/java/com/tc/l2/state/StateManagerImpl.java

```diff
@@ -336,12 +341,13 @@ public synchronized void moveToPassiveStandbyState() {
   }
 
   private synchronized void moveToActiveState() {
+    synchronizedWaitForStart();
```

Comment:
@myronkscott any particular reason for not using `synchronizedWaitForStart()` in `moveToActiveState()`, all other state transition methods have this safety check?